website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,101 @@
+
+1. Create a [deploy job](/docs/deploy/deploy-jobs) in dbt Cloud.
+2. Add the `--include-saved-query` to the `dbt build` command to re-run any Export that needs to be refreshed after a model is build. You can use the selector syntax i.e `--select` to only run the Exports downstream of a particular model.
+3. After dbt completes building the models, MetricFlow Server processes the Exports, compiles the necessary SQL, and executes this SQL against your data platform.
+4. Review the Exports execution details in the jobs logs and confirm the Export was run succesfully. Since Saved Queries are integrated into the dbt DAG, all outputs related to Exports are available in the job logs.
+5. Your data is now available in the data platform for querying.
+
+You can use the selector syntax i.e `--select`, `-s` to specify a praticular dbt model to run in your build command in order to only run the Exports downstream of that model. For example: `dbt build --include-saved-query --select orders+`. This will run any Saved Queries that are downstream of the orders semantic model.
+
+### Running Exports in Development
+Running an Export in development is useful if you want to test the output of the Export. You can run an Export in development using the cloud CLI command `dbt sl wxport` i.e `dbt sl wxport --saved-query sq_name`. This will run any Export defined for the Saved Query and materialize the table or view in your development environment.
+
+By default all Exports are run for a Saved Query. You can select a specific Export with the `select` flag or you can specify a new Export using the `export-as` flag. The Job scheduler runs the equivalent of` dbt sl export --saved-query sq_name`.
+
+| Parameters | Type    | Required | Description    |
+| ------- | --------- | ---------- | ---------------- |
+| `saved-query` | String    | Required     | A name of a Saved Query that could be used.    |
+| `select` | List or String   | Optional    | Names of Exports to be selected from the Saved Query. |
+| `exclude` | String  | Optional    | Not the names of Exports to be selected from the Saved Query. |
+| `export-as` | String  | Optional    | Type of Export to create from the Export_as types available in the config. |
+| `schema` | String  | Optional    | Schema to use for creating the table or view. |
+| `alias` | String  | Optional    | Table alias to use to write the table or view. |
+
+Since `--select` is used to select an existing Export, it can't be used with alias or schema to override  the Export configurations. If you want to override the Export configurations, you can use the `--export-as` flag along with the `--schema` and `--alias` flags. For example:
```

Comment:
can we get a screenshot of the input and output?

Reply:
Yup will add tomorrow.